react/Wrapper.tsx

```diff
@@ -61,14 +62,15 @@ const Wrapper: FC<Props> = ({
   customToastUrl,
   showToast,
   customOneClickBuyLink,
+  selectedSeller
 }) => {
   const productContext: ProductContextState = useProduct()
   const isEmptyContext = Object.keys(productContext).length === 0
 
   const product = productContext?.product
   const selectedItem = productContext?.selectedItem
   const assemblyOptions = productContext?.assemblyOptions
-  const selectedSeller = productContext?.selectedItem?.sellers[0]
+  selectedSeller = selectedSeller ? selectedSeller : productContext?.selectedItem?.sellers[0]
```

Comment:
Instead of changing the value of the received prop, keep the resolved value in a new variable. Something like:

```tsx
const seller = selectedSeller ?? productContext?.selectedItem?.sellers[0]
```

Reply:
Or you could rename the received value directly from the `props`, such as:
```tsx
const Wrapper: FC<Props> = ({
  isOneClickBuy,	
  available,
  disabled
  customToastUrl
  showToast
  customOneClickBuyLink
  selectedSeller: selectedSellerProp
})
```

Reply:
I personally tend to prefer the former suggestion of storing this in a new variable.

Reply:
In the case of the first suggestion, it will generate something similar to the second in another block of code

```
const skuItems = useMemo(
    () =>
      mapCatalogItemToCart({
        product,
        selectedItem,
        selectedQuantity,
        selectedSeller:seller,
        assemblyOptions,
      }),
    [assemblyOptions, product, selectedItem, selectedQuantity, seller]
  )
```

Reply:
I think that's fine, it wouldn't hurt readability 🙂

Reply:
Agreed, already commited the change.